### PR TITLE
Tag MXNet v0.2.0 [https://github.com/dmlc/MXNet.jl]

### PR DESCRIPTION
**Pull request by vchuravy**

### CODE REVIEW

File: MXNet/versions/0.2.0/requires [hunk 1]
```diff
@@ -0,0 +1,5 @@
+julia 0.5
+Compat 0.9.1
```
Comment (tkelman) [inline MXNet/versions/0.2.0/requires]: `take!` requires 0.9.5

**Comment by tkelman** [comment]
you have `BaseTestNext` in a lot of test files (under an if that won't get hit since you're 0.5-only now), but it wasn't listed in `test/REQUIRE`

**Comment by vchuravy** [comment]
Just addressed your comments. Should I delete the tag and release a new one or include these changes in an immediate `v0.2.1`?

**Comment by tkelman** [comment]
tightening version requirements needs to be done in place

**Pull request closed without merging**